include/oneapi/mkl/blas/detail/mklcpu/blas_ct.hpp

```diff
@@ -47,7 +48,7 @@ void syr2<library::intelmkl, backend::intelcpu>(cl::sycl::queue &queue, uplo upp
                                                 cl::sycl::buffer<float, 1> &y, std::int64_t incy,
                                                 cl::sycl::buffer<float, 1> &a, std::int64_t lda) {
     syr2_precondition(queue, upper_lower, n, alpha, x, incx, y, incy, a, lda);
-    onemkl::mklcpu::syr2(queue, upper_lower, n, alpha, x, incx, y, incy, a, lda);
+    oneapi::mkl::mklcpu::syr2(queue, upper_lower, n, alpha, x, incx, y, incy, a, lda);
```

Comment:
I think, the new direction is to use external/public APIs from Intel oneMKL. In that case, we should think how to handle mklcpu/mklgpu namespaces.

Reply:
Let's bring this up in a proposal later.